AOJ/06/06_B.py

```diff
@@ -0,0 +1,30 @@
+n = int(input())
+group = []
+while n != 0:
+    a,b1 = input().split()
+    b = int(b1)
+    group += [(a,b)]
+    n -= 1
```

Comment:
`for`ループでのリストに格納することができます。
- `while`のループ回数が`n`と決まっているため、`for`ループで書けます。
- `a`と`b`を別々で受けとって、再度タプルにまとめている処理はなくても大丈夫です。
- `int`処理は、続くコードで処理をした方が良いため、削除しました。
```suggestion
group = [(input().split()) for i in range(n)]
```